dev/com.ibm.ws.jsfContainer_fat_2.3/fat/src/com/ibm/ws/jsf/container/fat/tests/JSF22StatelessViewTests.java

```diff
@@ -85,16 +85,9 @@ public static void testCleanup() throws Exception {
     @Test
     public void verifyAppProviders() throws Exception {
         server.resetLogMarks();
-        server.waitForStringInLogUsingMark("Initializing Mojarra .* for context '/" + MOJARRA_APP + "'");
+        assertNotNull(server.waitForStringInLogUsingMark("Initializing Mojarra .* for context '/" + MOJARRA_APP + "'"));
         server.resetLogMarks();
-
-        String msgToSearchFor = "MyFaces CDI support enabled";
-
-        if (JakartaEE9Action.isActive()) {
-            msgToSearchFor = "MyFaces Core CDI support enabled";
-        }
-
-        server.waitForStringInLogUsingMark(msgToSearchFor);
+        assertNotNull(server.waitForStringInLogUsingMark("MyFaces Core CDI support enabled"));
```

Comment:
Since the personal build passed, the strings previously used were incorrect then?  `MyFaces Core CDI` vs `MyFaces CDI`